src/components/button/AccentButton.tsx

```diff
@@ -9,10 +9,10 @@ export enum AccentColor {
 }
 
 export const AccentButton = styled(Button)<{ color: AccentColor }>`
-  background: ${p => colorForOption(p.color)};
+  background: ${(p) => colorForOption(p.color)};
```

Comment:
Small note: it's usually better to do a separate PR for unrelated changes to avoid "polluting" it.

Reply:
This was totally prettier sneaking in a change and not me. I 100% agree this kind of thing should be separate. It's not the first time prettier has tried this 🙃

Reply:
There us a button "Restore file" in the top right of the file here in Github UI where you can undo changes for individual files (at least if you have Refined Github installed).

Reply:
Note: Restore file thing doesn't work for me anymore recently (fails) so 🤷🏼‍♂️

Reply:
Then it can still be done easily in the branch to get a clean PR and not introduce changes, that someone else with a different prettier config (or whatever is causing this) will undo _again_.

Reply:
We all should have the same prettier config, Its set in the `.perttierrc` file. I might have a global config that I haven't removed that could be interfering.